aquarius/app/assets.py

```diff
@@ -741,3 +743,12 @@ def _validate_date_format(date):
 def _my_converter(o):
     if isinstance(o, datetime):
         return o.strftime('%Y-%m-%dT%H:%M:%SZ')
+
+
+def _make_paginate_response(query_list_result, search_model):
+    result = dict()
+    result['result'] = query_list_result
```

Comment:
Let's make it plural:

```suggestion
    result['results'] = query_list_result
```